src/lib/y2network/connection_config/bridge.rb

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/connection_config/base"
+
+module Y2Network
+  module ConnectionConfig
+    # Configuration for bridge connections
+    #
+    # @see https://www.kernel.org/doc/Documentation/networking/bridge.txt
+    class Bridge < Base
+      # @return [Array<Interface>]
+      attr_accessor :ports
+      # @return [Boolean] whether spanning tree protocol is enabled or not
+      attr_accessor :stp
+      # @return [Integer]
+      attr_accessor :forward_delay
+
+      def initialize
+        @ports = []
+        @stp = false
+        @forward_delay = 0
+      end
```

Comment:
so here ports cannot be fake ones?

Reply:
Why not?, Them are interfaces and a Fake interface is an Interface.